packages/amesos/test/TestOptions/CreateTridi.cpp

```diff
@@ -86,7 +91,9 @@ int CreateTridi(Epetra_CrsMatrix& A)
 	Values[0] = -1.0; 
 	Values[1] = 2.0;
 	Values[2] = -1.0;
+#ifndef NDEBUG
 	NumEntries = 3;
+#endif
       }
     
     assert(A.InsertGlobalValues(MyGlobalElements[i], NumEntries, Values, Indices)==0);
```

Comment:
Wow, this test really only does anything in a debug build.  @trilinos/amesos @trilinos/amesos2 did y'all realize this?

@ZUUL42 , it's a bit out of scope to try to fix this issue -- I'm just pointing it out to the Amesos developers.